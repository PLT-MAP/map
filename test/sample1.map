func main(){
        Graph g=new Graph();
        Node nj= new Node();
        Node ny= new Node();
        Node pa= new Node({'temp':85,'humidity':'low'});
	Node va= new Node({'temp':87,'humidity':'high'});
}














