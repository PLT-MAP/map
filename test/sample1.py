import networkx as nx
def main():	
	g=nx.Graph()
	nj = "nj"
	ny = "ny"
	pa = "pa"
	padict= {'temp':85,'humidity':'low'}
	va = "va"
	vadict= {'temp':87,'humidity':'high'}
	
if __name__== '__main__':
	main()




