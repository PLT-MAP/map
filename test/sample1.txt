def main():
        g=nx.MultiDiGraph()
        nj = 'nj'
        ny = 'ny'
        pa = 'pa', {'temp':85,'humidity':'low'}
        va = 'va', {'temp':87,'humidity':'high'}
