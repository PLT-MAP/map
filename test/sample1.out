import networkx as nx
import sys
import matplotlib.pyplot as plt
def main():
    g = nx.MultiDiGraph()
    nj = 'nj',{}
    ny = 'ny',{}
    pa = 'pa',  {'temp':85.0,'humidity':'low'}
    va = 'va',  {'temp':87.0,'humidity':'high'}
if __name__ == '__main__': 
	try:
		main()
	except:
		print'Error:',sys.exc_info()[0]
		print'Resolve error before running again!'